src/router/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Route, Router, IndexRoute, hashHistory } from 'react-router';
+
+import App from '../components/App/App';
+
+export default (
+  <Router history={hashHistory}>
```

Comment:
we cannot use the hash...we need full url paths, with browser support to be able to render well on serve.